portal/__init__.py

```diff
@@ -44,10 +44,12 @@ def create_app(test_config=None):
 
     # Register Routes
     # ---------------
-    @app.route('/')
-    def index():
-        return render_template('index.html')
+    from . import college
+    app.register_blueprint(college.bp)
```

Comment:
The whole portal is for the college. Is there a better or more specific name for this module?